apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -391,21 +391,21 @@
                             "required" : [
                                 "executable",
                                 "extra_args",
-                                "node_bits"
+                                "edge_bits"
                             ],
                             "properties" : {
                                 "executable" : {
-                                    "description" : "Executable binary of the miner. Options are: \"mean30s-generic\" (memory-intensive), \"mean30s-avx2\" (memory-intensive, benefits from faster CPU supporting AVX2 instructions), \"lean30\" (CPU-intensive, useful if memory-constrained).",
+                                    "description" : "Executable binary of the miner. Options are: \"mean29-generic\" (memory-intensive), \"mean29-avx2\" (memory-intensive, benefits from faster CPU supporting AVX2 instructions), \"lean29-generic\" (CPU-intensive, useful if memory-constrained), \"lean29-avx2\" (CPU-intensive, useful if memory-constrained, benefits from faster CPU supporting AVX2 instructions).",
```

Comment:
Will require update to https://github.com/aeternity/epoch/wiki/Troubleshooting#node-failing-mining-attempts